src/components/CurrentUser/CurrentUser.jsx

```diff
@@ -1,12 +1,53 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './CurrentUser.scss';
+import { request } from '../../api/api';
 
-export const CurrentUser = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+export class CurrentUser extends React.Component {
+  state = {
+    user: {},
+  }
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+  componentDidMount() {
+    request('/users', `/${this.props.userId}`)
+      .then(({ data }) => this.setState({ user: { ...data } }));
```

Comment:
When you write in url userId you receive user obj, not data. So you can make it simpler. Like: _.then(user => this.setState({ user }) )_

Reply:
it doesn't work)

Reply:
Destructuring is used here, so it needs exact property name to work correctly.